data.sql

```diff
@@ -62,4 +62,187 @@ UPDATE animals
 SET owners_id = (
     SELECT id FROM owners WHERE full_name = 'Dean Winchester'
 )
-WHERE name IN ('Angemon', 'Boarmon');
\ No newline at end of file
+WHERE name IN ('Angemon', 'Boarmon');
+
+
+
+
+
+INSERT INTO vets (name, age, date_of_graduation)
```

Comment:
- [ ] To meet up with the requirement and to avoid anomalies, I think it is best to follow the specifics of the project as shown below:
![image](https://github.com/Xghayor/vet-clinic-db/assets/99715640/2bd1cd92-2876-47b7-ad31-c53562b745bf)

- [ ]  You are missing the following:
  - [ ] `id` with auto increment and primary key set on it
  - [ ] declared `types` of the entity to protect the database against unwanted data types. examples are: `int`, `date`, `string`, etc.

Reply:
Hi @AbayomiOlaoye ,
Thanks for the feedback, However, did you check the code?
![6](https://github.com/Xghayor/vet-clinic-db/assets/122298629/b86d8313-ae34-4a89-a1d0-0ec7e5f604ca)
Kindly have a look!

Reply:
Oh, you are right! Sorry about that

Reply:
Al'right, i have submitted again